modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc

```diff
@@ -438,3 +438,22 @@ PostgreSQL: \c owncloud; \d
 MySQL     : quit
 PostgreSQL: \q
 ....
+
+[[how-to-solve-deadlock-error]]
```

Comment:
This explicit anchor/id isn't required, as one exactly the same will be auto-generated.